tests/event_test_utils.rs

```diff
@@ -0,0 +1,96 @@
+use rust_q_sim::config::Config;
```

Comment:
Ah, very good. This will become more elaborate I guess. Also numbers like time, and others shouldn't be strings I guess. They will be converted by serde automatically. 

I am planning to create an events specification via protobuf as part of #26. I don't know yet how much overlap this is.

Reply:
With the numbers it was a bit odd - the auto conversion didn't work for some reason. I will try it out again.